src/test/java/domain/RaceProgressTest.java

```diff
@@ -0,0 +1,26 @@
+package domain;
+
+import domain.car.Cars;
+import domain.name.Name;
+import domain.name.Names;
+import domain.race.RaceProgress;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class RaceProgressTest {
+    @Test
+    @DisplayName("자동차 목록 포함한 일급객체를 통해 자동차 결과를 포함한 일급객체를 만든다.")
```

Comment:
```suggestion
    @DisplayName("자동차 목록 포함한 일급객체를 통해 자동차 결과를 포함한 일급컬렉션 만든다.")
```
일급 객체(First Class Object)는 다르게 사용되는 용어입니다.

Reply:
감사합니다🙇‍♂️